modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/action/executor/FunctionActionExecutorImpl.java

```diff
@@ -171,42 +172,47 @@ else if (jsonObject.has("serializable")) {
 					"There are no workflow task assignees for Kaleo action " +
```

Comment:
Do we still need this condition just to log this? Maybe removing this `return;` statement will make this unusable.